### PR TITLE
Remove PNS ID

### LINKED ISSUE #154: Only use either Path ID or PNS ID
From PR #149 from @qdeconinck:

> Maybe it can be addressed later, but given that the Path Identifier and the PNS ID are now always the same, we may simplify this section and only introduce one concept (i.e., the Path Identifier).

### PR DESCRIPTION
**Pull request by mirjak**
fixes #154

### CODE REVIEW

File: draft-ietf-quic-multipath.md [hunk 1]
```diff
@@ -172,29 +172,12 @@ We assume that the reader is familiar with the terminology used in
...
-
-The initial path that is used during the handshake (and multipath negotiation)
-has the path ID 0 and therefore all 0-RTT packets are also tracked and
-processed with the path ID 0.
-For 1-RTT packets, the path ID is the sequence number of
-the Destination Connection ID present in the packet header, as defined
-in {{Section 5.1.1 of QUIC-TRANSPORT}}.
+  a path. The initial path that is used during the handshake (and multipath negotiation)
+  has the path ID 0 and therefore all 0-RTT packets are also tracked and
+  processed with the path ID 0. For 1-RTT packets, the path ID is the
```
Comment (mirjak) [inline draft-ietf-quic-multipath.md:177]:
```suggestion
  processed with the path ID 0. For 1-RTT packets, the path ID is the
```

File: draft-ietf-quic-multipath.md [hunk 2]
```diff
@@ -172,29 +172,12 @@ We assume that the reader is familiar with the terminology used in
 - Path Identifier (Path ID): An identifier that is used to identify
   a path in a QUIC connection at an endpoint. Path Identifier is used
   in multipath control frames (etc. PATH_ABANDON frame) to identify
```
Comment (qdeconinck) [inline draft-ietf-quic-multipath.md:174]:
```suggestion
  in multipath control frames (e.g., PATH_ABANDON frame) to identify
```

File: draft-ietf-quic-multipath.md [hunk 3]
```diff
@@ -587,12 +570,12 @@ the ACK frame as well as packet protection as described in the following subsect
 
 The ACK_MP frame, as specified in {{ack-mp-frame}}, is used to
 acknowledge 1-RTT packets.
-Compared to the QUIC version 1 ACK frame, the ACK_MP frames additionally
-contains a Packet Number Space Identifier (PN Space ID).
-The PN Space ID used to distinguish packet number spaces for different
+Compared to the QUIC version 1 ACK frame, the ACK_MP frame additionally
+contains a Path ID.
+The Path ID used to distinguish packet number spaces for different
```
Comment (qdeconinck) [inline draft-ietf-quic-multipath.md:575]:
```suggestion
The Path ID is used to distinguish packet number spaces for different
```

**Review by qdeconinck — APPROVED** [review]
Sounds good.

**Review by Yanmei-Liu — APPROVED** [review]
The conflict was solved. LGTM.

**Pull request merged**